test/e2e/nns_update_timestamp_test.go

```diff
@@ -0,0 +1,36 @@
+package e2e
+
+import (
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("NNS LastSuccessfulUpdate", func() {
+	Context("when updating nns", func() {
+		It("timestamp should be changed", func() {
+			for _, node := range nodes {
+				var interval int = 0
+				key := types.NamespacedName{Name: node}
+				timeBefore := nodeNetworkState(key).Status.LastSuccessfulUpdate
+
+				configMap, err := framework.Global.KubeClient.CoreV1().ConfigMaps("nmstate").Get("nmstate-config", metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				interval, err = strconv.Atoi(configMap.Data["node_network_state_refresh_interval"])
+				Expect(err).ToNot(HaveOccurred())
+				interval = interval * 3
+				ReadTimeout := time.Duration(interval) * time.Second
+
+				Eventually(func() bool {
+					timeAtfer := nodeNetworkState(key).Status.LastSuccessfulUpdate
+					return timeAtfer.Time.After(timeBefore.Time)
+				}, ReadTimeout, ReadInterval).Should(BeTrue())
```

Comment:
There is a Should method that is meant for time comparison. It would give us better message in case of a failure.

https://onsi.github.io/gomega/#betemporallycomparator-string-compareto-timetime-threshold-timeduration